ios/RNSScreenStack.mm

```diff
@@ -801,6 +801,7 @@ - (void)navigationController:(UINavigationController *)navigationController
 {
   [self emitOnFinishTransitioningEvent];
   [RNSScreenWindowTraits updateWindowTraits];
+  [_controller.view setNeedsLayout];
```

Comment:
isn't the `_controller.view == self`?

Reply:
Not quite - as I'm looking onto `_controller.view` it's UILayoutContainerView, whereas self is just RNSScreenStackView